### ISSUE TITLE
Bad useability of "open zim file" dialog on small screens

### ISSUE THREAD

**Issue by cip** [cip/WikiOnBoard#7]
Uses QT QFileDialog which is not exactly perfect for small screens with low resolutions.
Fixing this is low priority, because main use case is that dialog is used only once to select the zim file (wikipedia)  and then not changed frequently later.
As soon as  a larger number of zim file format eBooks are available it will be more important to solve this issue. One  relative simple possibility would be to keep the dialog for file selection, but to additionally implement functionality which allows switching between recently used files.

**Comment by cip** [comment]
On Symbian 3 Qt4.7 problem is fixed, qt automatically uses native file dialog.Unfortunately  this does not apply for older Symbian devices (and therefore all current non-touch devices) .
